torch/_torch_docs.py

```diff
@@ -5510,6 +5510,9 @@ def merge_dicts(*dicts):
 .. note:: The returned tensor shares the storage with the input tensor,
           so changing the contents of one will change the contents of the other.
 
+.. warning:: If batch_size=1, then `squeeze(input)` will also remove the batch
```

Comment:
Not all tensors have something called a "batch_size". Could we write something like "if the tensor has a batch dimension of size 1" ?